sdks/python/apache_beam/runners/portability/spark_runner.py

```diff
@@ -49,7 +53,15 @@ def default_job_server(self, options):
         raise ValueError('Option spark_rest_url must be set.')
       return spark_uber_jar_job_server.SparkUberJarJobServer(
           spark_options.spark_rest_url, options)
-    return job_server.StopOnExitJobServer(SparkJarJobServer(options))
+    # Use Java job server by default.
+    # Only SparkRunnerOptions and JobServerOptions affect job server
```

Comment:
How sure are we that this will remain true in the future?

Reply:
I added an assertion for that in the test.